java/java.project.ui/nbproject/org-netbeans-modules-java-project-ui.sig

```diff
@@ -268,6 +268,13 @@ CLSS public abstract interface static org.netbeans.spi.java.project.support.ui.B
 meth public abstract void platformPropertyUpdated(org.netbeans.api.java.platform.JavaPlatform)
  anno 1 org.netbeans.api.annotations.common.NonNull()
 
+CLSS public org.netbeans.spi.java.project.support.ui.CreateJavaClassFileFromClipboard
```

Comment:
Result of 56f16fb193c - we probably don't want this `CreateJavaClassFileFromClipboard extends PasteType` class to be visible in the API. Can you move it to implementation package, @singh-akhilesh?